frontend/src/metabase/embedding-sdk/theme/default-component-theme.ts

```diff
@@ -88,6 +89,9 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
       label: { fontSize: FONT_SIZES.goalLabel.px },
     },
   },
+  popover: {
+    zIndex: DEFAULT_Z_INDEX,
+  },
```

Comment:
This makes the `--mb-overlay-z-index` variable defaults to DEFAULT_Z_INDEX, which is `4`